src/main/java/ru/testtask/Application.java

```diff
@@ -3,19 +3,31 @@
 
 import org.modelmapper.ModelMapper;
 import org.modelmapper.convention.MatchingStrategies;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import ru.testtask.service.UserService;
+
+import javax.annotation.PostConstruct;
 
 import static org.modelmapper.config.Configuration.AccessLevel.PRIVATE;
 
 
 @SpringBootApplication(exclude = {DataSourceAutoConfiguration.class})
 @EnableAspectJAutoProxy
 public class Application {
+    @Autowired
+    private UserService userService;
+
     public static void main(String[] args) {
         SpringApplication.run(Application.class, args);
     }
+
+    @PostConstruct
+    public void createUsers(){
+        userService.createDefaultUsers();
```

Comment:
It is better to move this method to `UserService` or just add `@PostConstruct` to `UserService#createDefaultUsers()`